faq.md

```diff
@@ -0,0 +1,139 @@
+
+Please read the installation instructions for the appropriate distribution:
+- [Debian installation instructions](running.html#debian)
+- [Ubuntu installation instructions](running.html#ubuntu)
+
+### How do I use Cockpit with a proxy?
+
+Cockpit can be used from behind a proxy.
+
+Read one of the following documents:
+
+- [Proxying Cockpit over NGINX](external/wiki/Proxying-Cockpit-over-NGINX.html)
+- [Proxying Cockpit over Apache with LetsEncrypt](external/wiki/Proxying-Cockpit-over-Apache-with-LetsEncrypt.html)
+
+## Development
+
+### How do I build an app for Cockpit?
+
+- [Starter Kiit](blog/cockpit-starter-kit.html)
+- [Make your Cockpit page easily installable](blog/making-a-cockpit-application.html)
+- [Additional developer-related resourcesare on the Contributing page](external/wiki/Contributing.html)
+
+## Troubleshooting
+
+### Login
+
+#### Blank white page after login
+
+##### Blank page with a Proxy
+
+You may be using Cockpit with an improperly set up proxy.
+
+###### Solutions
+{:.no_toc}
+
+- [Proxying Cockpit over NGINX](external/wiki/Proxying-Cockpit-over-NGINX.html)
+- [Proxying Cockpit over Apache with LetsEncrypt](external/wiki/Proxying-Cockpit-over-Apache-with-LetsEncrypt.html)
+
+##### Blank page using Safari
+
+Older versions of Safari on iPhones, iPads, and Safari on macOS on M1 (ARM) machines have a problem with encrypted WebSocket.
+
+###### Solution
+{:.no_toc}
+
+Newer updates of Safari no longer have this issue, but some older [devices may need to use a workaround](running/safari.html).
+
+### Software update
+
+#### Error message about "OSTree"
+
+You're seeing "OSTree is not available on this system" or "Unable to communicate with OSTree" instead of a software update page.
+
+In all liklihood, this means you have `cockpit-ostree` installed on a system that does not use `rpm-ostree` to manage its packages. You should be using Cockpit's built-in PackageKit-based software updater, not `cockpit-ostree`.
+
+##### Solution
+{:.no_toc}
+
+Remove the ostree package:
+
+```
+dnf remove cockpit-ostree
+```
+
+`cockpit-ostree` is only intended for use on distributions that use `rpm-ostree`, such as Fedora Silverblue, Fedora Kinoite, Fedora IoT, Fedora CoreOS, and a small subset of Red Hat Enterprise Linux that uses `rpm-ostree`.
+
+#### Error message about being offline
+
+The software update page shows "packagekit cannot refresh cache whilst offline" on a Debian or Ubuntu system.
+
+##### Solution
+{:.no_toc}
+
+Create a placeholder file and network interface.
+
+1. Create `/etc/NetworkManager/conf.d/10-globally-managed-devices.conf` with the contents:
+
+    ```ini
+    [keyfile]
+    unmanaged-devices=none
+    ```
+
+2. Set up a "dummy" network interface:
+
+    ```
+    nmcli con add type dummy con-name fake ifname fake0 ip4 1.2.3.4/24 gw4 1.2.3.1
+    ```
+
+3. Reboot
+
+##### Explanation
+{:.no_toc}
+
+Both Debian and Ubuntu use two different network stacks which don't play so well together.
```

Comment:
Debian does not use netplan by default, and it's unlikely that people use it. People could explicitly configure networkd, though, but that's equally true on Fedora and all other distros. So I'd perhaps leave this out and restrict this section to Ubuntu/netplan?